src/data-table/_data-table.scss

```diff
@@ -24,6 +24,7 @@
   white-space: nowrap;
   font-size: 13px;
   background-color: unquote("rgb(#{$color-white})");
+  -webkit-transform: translate3d(0,0,0);
```

Comment:
I disagree; this is a Safari-specific workaround, so it should be prefixed to limit the number of browsers affected by it, and having potential side effects.

That said, I don't want to go back and forth on this, so I'll leave it up to @PatrickOgilvie to decide what to do here.